test/IntegrationTests/Helpers/TestHttpServer.NetFramework.cs

```diff
@@ -61,18 +61,25 @@ public TestHttpServer(ITestOutputHelper output, Action<HttpListenerContext> requ
                 _listenerThread.Start();
                 WriteOutput($"Listening on '{_prefix}'");
 
+                // call healthz (wait until HttpListener is operational)
+                var healhtzEndpoint = $"{_prefix.Replace("*", "localhost")}/healthz";
```

Comment:
Might not be needed any more at all. Since the bug happens with .NET Core, but .NET Core has now different HttpServer base.

Reply:
I will make a PR to remove it and run all tests a lot of times to make sure it is stable.